crates/starknet-devnet/tests/test_abort_blocks.rs

```diff
@@ -0,0 +1,287 @@
+pub mod common;
+
+mod abort_blocks_tests {
+    use hyper::Body;
+    use serde_json::json;
+    use server::api::json_rpc::error::ApiError;
+    use starknet_rs_core::types::{FieldElement, TransactionStatus};
+    use starknet_rs_providers::Provider;
+    use starknet_types::rpc::transaction_receipt::FeeUnit;
+
+    use crate::common::background_devnet::BackgroundDevnet;
+    use crate::common::utils::get_json_body;
+
+    static DUMMY_ADDRESS: u128 = 1;
+    static DUMMY_AMOUNT: u128 = 1;
+
+    #[tokio::test]
+    async fn abort_latest_block() {
+        let devnet: BackgroundDevnet =
+            BackgroundDevnet::spawn_with_additional_args(&["--state-archive-capacity", "full"])
+                .await
+                .expect("Could not start Devnet");
+
+        devnet.post_json("/create_block".into(), Body::from(json!({}).to_string())).await.unwrap();
+        let first_block = &devnet
+            .send_custom_rpc("starknet_getBlockWithTxHashes", json!({ "block_id": "latest" }))
+            .await["result"];
+
+        devnet.post_json("/create_block".into(), Body::from(json!({}).to_string())).await.unwrap();
+        let second_block = &devnet
+            .send_custom_rpc("starknet_getBlockWithTxHashes", json!({ "block_id": "latest" }))
+            .await["result"];
+
+        let abort_blocks = devnet
+            .post_json(
+                "/abort_blocks".into(),
+                Body::from(json!({ "startingBlockHash": second_block["block_hash"] }).to_string()),
+            )
+            .await
+            .unwrap();
```

Comment:
Add a helper function in this file that receives a block hash, aborts and returns the aborted vector

Reply:
Fixed in 684d7eae8cff553b04ea417c2c311a7bbf9cac31